packages/@uppy/transloadit/src/AssemblyOptions.js

```diff
@@ -105,7 +107,7 @@ class AssemblyOptions {
 
     if (options.alwaysRunAssembly) {
       // No files, just generate one Assembly
-      const assemblyOptions = await options.getAssemblyOptions(null, options)
+      const assemblyOptions = await options.assemblyOptions(null, options)
```

Comment:
shouldn't we use the same logic here as above?

Reply:
I can't call `this.#getAssemblyOptions` as that is made to stop when there is no file, but here we do want to run the file. But you're right I should check what the value is.